keyboards/draytronics/scarlet/keymaps/default/keymap.c

```diff
@@ -0,0 +1,28 @@
+/*Copyright 2020 Blake Drayson / Draytronics
+
+Contact info@draytronics.co.uk
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_numpad_5x4(
+      KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
+      KC_P7,   KC_P8,   KC_P9, KC_PPLS,
+      KC_P4,   KC_P5,   KC_P6,
+      KC_P1,   KC_P2,   KC_P3, KC_PENT,
+      KC_P0,   KC_PDOT)
```

Comment:
You have reverted this back to the wrong order

```suggestion
      KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
      KC_P7,   KC_P8,   KC_P9,
      KC_P4,   KC_P5,   KC_P6, KC_PPLS,
      KC_P1,   KC_P2,   KC_P3,
      KC_P0,   KC_PDOT, KC_PENT)
```

Reply:
Apologies, this failed to compile and I understand the mistake now, the physical design of the PCB has row two with four buttons and row four with four buttons.  So it goes, 4,4,3,4,2.  I need to revert the change that was made to scarlet.h as well.